src/planscape/forsys/scenario_sets.R

```diff
@@ -2,21 +2,24 @@ library(dplyr)
 library(forsys)
 library(sf)
 
-scenario_sets <- function(input_stand_data, priorities) {
-  input_stand_data
+scenario_sets <- function(input_stand_data, priorities, stand_id_field,
+                          proj_id_field, stand_area_field, stand_cost_field) {
+
+  scenario_output_fields <- append(priorities, c(stand_area_field, stand_cost_field))
 
   stand_dat <- test_forest %>% st_drop_geometry()
   run_outputs <- forsys::run(
     return_outputs = TRUE,
-    scenario_name = "test_scenario",
+    write_outputs = FALSE,
     stand_data = input_stand_data,
-    stand_id_field = "stand_id",
-    proj_id_field = "proj_id",
-    stand_area_field = "area",
+    stand_id_field = stand_id_field,
+    proj_id_field = proj_id_field,
+    stand_area_field = stand_area_field,
     scenario_priorities = priorities,
-    scenario_weighting_values = c("1 5 1"), 
+    scenario_weighting_values = c("1 5 1"),
+    scenario_output_fields = scenario_output_fields, 
     proj_fixed_target =  FALSE,
-    proj_target_field = "area",
+    proj_target_field = stand_area_field,
     proj_target_value = 1.0
```

Comment:
Does `proj_target_value` need to be passed on from the front end as well?

Reply:
We want to constrain by cost and area but Forsys can only constrain by one variable.
Setting proj_fixed_target=FALSE and proj_target_value=1.0 is a roundabout way to rank all project areas.
proj_target_value is a little arbitrary, but it must have a valid value - setting it to either stand_area_field or stand_cost_field rather than passing a new variable makes the django-R interface simpler.